pkg/mongoproxy/plugins/schema/schema.go

```diff
@@ -28,11 +28,15 @@ var (
 		Name: "mongoproxy_plugins_schema_config_hash",
 		Help: "The current hash of the schema config file loaded",
 	})
-
 	schemaDeny = promauto.NewCounterVec(prometheus.CounterOpts{
 		Name: "mongoproxy_plugins_schema_deny_total",
 		Help: "The total deny returns of a command",
 	}, []string{"db", "collection", "command"})
+
+	collectionSchemaLogOnlyDeny = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Minor: can we change metrics name to `schemaDenyLogOnly`.
When we have logonly deny, at our side we dont care about it's collection level or global level, and it has shorter names 😉

Reply:
otherwise LGTM!